R/data.R

```diff
@@ -499,7 +520,10 @@ spim_lancelot_data_rtm <- function(date, region, model_type, data, full_data,
     data$pillar2_negatives <- c(data$negatives[-c(1, 2)], rep(NA_integer_, 2))
     data$pillar2_positives_over25 <- data$positives_over25
     ## We do not have any age breakdown for negatives for Scotland
-    data$pillar2_negatives_over25 <- NA_integer_
+    for (i in c("over25", pillar2_age_bands)) {
```

Comment:
Including "over25" here is overwriting the positives over 25 (which we have for Scotland. Remove this and reinstate the deleted line above

Reply:
Since we have over 25 positives but not negatives for Scotland, I would think that would mean we could fit by age for the NB for Scotland but not the BB

Reply:
Thanks, corrected this now